cross/pip/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME = pip
 # use the same version in native/python3/Makefile (python3_native_post_install)
-PKG_VERS = 20.2.4
+PKG_VERS = 21.1.1
```

Comment:
@smaarn The comment here is the reason for my update in native/python3/Makefile

Reply:
I see.
The only concern I have here is that this means this update impacts all python3 dependent packages. Even though it should be ok, shouldn't we have a "pip3" specialized cross package ? I do understand anyway it would really make this PR much heavier (potential side effects ...)

Reply:
We have a pip3 cross package the `cross/pip`. For python 2.7 we have `cross/pip_py2` as pip >= 21.0 is for python >= 3.